cfme/tests/automate/custom_button/test_rest_custom_button.py

```diff
@@ -18,10 +20,29 @@
     pytest.mark.uncollectif(
         lambda appliance, obj_type: obj_type not in OBJ_TYPE_59 and appliance.version < "5.10"
     ),
-    pytest.mark.parametrize("obj_type", OBJ_TYPE, ids=[obj.capitalize() for obj in OBJ_TYPE]),
+    pytest.mark.parametrize("obj_type", OBJ_TYPE, ids=[obj.capitalize() for obj in OBJ_TYPE],
+                            scope="module"),
 ]
 
 
+@pytest.fixture(scope="module")
+def group(request, appliance, obj_type):
```

Comment:
Might want to be more explicit in the naming here, since this is apparently returning a group REST ID, same below.

Reply:
not rest id actually but rest entity so i put `_rest`